pkg/minikube/registry/drvs/hyperkit/hyperkit.go

```diff
@@ -85,5 +88,57 @@ func status() registry.State {
 		return registry.State{Installed: true, Error: fmt.Errorf("%s failed:\n%s", strings.Join(cmd.Args, " "), out), Fix: "Run 'brew install hyperkit'", Doc: docURL}
 	}
 
+	// Split version from v0.YYYYMMDD-HH-xxxxxxx or 0.YYYYMMDD to YYYYMMDD
+	specificVersion := splitHyperKitVersion(version.GetHyperKitVersion())
+	// If hyperkit is not newer than specificVersion, suggest upgrade information
+	isNew := isNewerVersion(convertVersionToDate(string(out)), specificVersion)
+	if !isNew {
+		return registry.State{Installed: true, Error: fmt.Errorf("%s is too old. the higher than %s is needed", "hyperkit version", version.GetHyperKitVersion()), Fix: "Run 'brew upgrade hyperkit' or Upgrade Docker for Desktop version", Doc: docURL}
```

Comment:
Lets make sure this is a non-fatal error and only warn the user ! as someone might not have corp access or not allowed to install a newer version.

Reply:
the error message could be rephrased to

"the mininum recommended hyperkit version is .... your version is .... please consider upgrading your hyperkit.
Fix : brew upgrade hyperkit 
for more information read ... (link to doc on minikube site)
"

Reply:
I fixed this to be non-fatal error and only warn the user.

```
⚠️   'hyperkit' driver reported a possible issue: the mininum recommended hyperkit version is 0.20190201. your hyperkit version is 0.20180621. please consider upgrading your hyperkit
💡  Suggestion: Run 'brew upgrade hyperkit'
📘  Documentation: https://minikube.sigs.k8s.io/docs/reference/drivers/hyperkit/
```